openedx_lti_tool_plugin/tests/test_utils.py

```diff
@@ -71,23 +127,30 @@ def test_without_available_sequences(
             datetime_mock: Mocked datetime function.
             get_user_course_outline_mock: Mocked get_user_course_outline function.
             get_outline_tree_mock: Mocked get_course_outline_block_tree function.
-            from_string_mock: Mocked CourseKey from_string method.
+            course_key_from_string_mock: Mocked CourseKey from_string method.
+            modulestore_mock: Mocked modulestore class.
         """
         get_user_course_outline_mock.return_value = MagicMock(sequences=[])
 
+        outline = get_course_outline(self.request_mock, COURSE_ID)
+
         self.assertEqual(
-            get_course_outline(self.request_mock, COURSE_ID),
+            outline,
```

Comment:
```suggestion
            get_course_outline(self.request_mock, COURSE_ID),
```
Same has previous comment